webpack/assets/javascripts/react_app/components/Bookmarks/components/SearchModal.js

```diff
@@ -7,20 +7,23 @@ import { noop } from '../../../common/helpers';
 import BookmarkForm from './BookmarkForm';
 
 const SearchModal = ({
+  id,
   setModalClosed,
   onEnter,
   title,
   controller,
   url,
   bookmarks,
+  isPF4,
 }) => (
   <ForemanModal
-    id={BOOKMARKS_MODAL}
+    id={isPF4 ? `${BOOKMARKS_MODAL}-${id}` : BOOKMARKS_MODAL}
```

Comment:
can you use the same helper to generate the PF4 modal id?

Also, is the PF3 bookmark used anywhere? thinking if we should drop it to simplify things.

Reply:
If it's not used we can remove it in a new pr

Reply:
added  a helper